composer.json

```diff
@@ -35,6 +35,7 @@
         "symfony/serializer": "^6.4 || ^7.0",
         "symfony/translation-contracts": "^3.3",
         "symfony/web-link": "^6.4 || ^7.0",
+        "symfony/yaml": "^6.4 || ^7.0",
```

Comment:
isn't that inside the suggestions? It's optional IIRC.

Reply:
Yes, now it is in the `suggest` node. However, the API Platform would not work without this.

You can see it here - https://github.com/eonx-com/easy-monorepo/actions/runs/9701214835/job/26774471651

If no access, logs below:
```
TypeError: ApiPlatform\Serializer\YamlEncoder::__construct(): Argument #2 ($yamlEncoder) must be of type Symfony\Component\Serializer\Encoder\EncoderInterface&Symfony\Component\Serializer\Encoder\DecoderInterface, null given, called in /home/runner/work/easy-monorepo/easy-monorepo/packages/EasyApiPlatform/tests/Fixture/app/var/cache/test/ContainerKxTdjbe/EonX_EasyApiPlatform_Tests_Fixture_App_Kernel_ApplicationKernelTestDebugContainer.php on line 1834

/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyApiPlatform/vendor/api-platform/core/src/Serializer/YamlEncoder.php:25
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyApiPlatform/tests/Fixture/app/var/cache/test/ContainerKxTdjbe/EonX_EasyApiPlatform_Tests_Fixture_App_Kernel_ApplicationKernelTestDebugContainer.php:1834
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyApiPlatform/tests/Fixture/app/var/cache/test/ContainerKxTdjbe/EonX_EasyApiPlatform_Tests_Fixture_App_Kernel_ApplicationKernelTestDebugContainer.php:1394
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyApiPlatform/tests/Fixture/app/var/cache/test/ContainerKxTdjbe/EonX_EasyApiPlatform_Tests_Fixture_App_Kernel_ApplicationKernelTestDebugContainer.php:744
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyApiPlatform/tests/Fixture/app/var/cache/test/ContainerKxTdjbe/EonX_EasyApiPlatform_Tests_Fixture_App_Kernel_ApplicationKernelTestDebugContainer.php:149
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyApiPlatform/vendor/symfony/event-dispatcher/EventDispatcher.php:243
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyApiPlatform/vendor/symfony/event-dispatcher/EventDispatcher.php:206
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyApiPlatform/vendor/symfony/event-dispatcher/EventDispatcher.php:56
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyApiPlatform/vendor/symfony/http-kernel/HttpKernel.php:157
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyApiPlatform/vendor/symfony/http-kernel/HttpKernel.php:76
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyApiPlatform/vendor/symfony/http-kernel/Kernel.php:197
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyApiPlatform/vendor/symfony/http-kernel/HttpKernelBrowser.php:65
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyApiPlatform/vendor/symfony/framework-bundle/KernelBrowser.php:175
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyApiPlatform/vendor/symfony/browser-kit/AbstractBrowser.php:385
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyApiPlatform/vendor/api-platform/core/src/Symfony/Bundle/Test/Client.php:116
/home/runner/work/easy-monorepo/easy-monorepo/packages/EasyApiPlatform/tests/Application/src/Paginator/CustomPaginatorTest.php:19

```

Reply:
The `\ApiPlatform\Serializer\YamlEncoder` service loaded in https://github.com/api-platform/core/blob/0d5f356839eb6aa9f536044abe4affa736553e76/src/Symfony/Bundle/Resources/config/openapi.xml#L103

And there are no checks here https://github.com/api-platform/core/blob/0d5f356839eb6aa9f536044abe4affa736553e76/src/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php#L528